bin/pdc_client

```diff
@@ -119,6 +119,9 @@ def load_data(options):
 if __name__ == "__main__":
     parser = optparse.OptionParser(version=__version__)
     parser.add_option("-s", "--server", help="PDC instance url or shortcut.")
+    parser.add_option("-k", "--insecure", action="store_true",
+                      help="Disable SSL certificate verification")
+    parser.add_option("--ca-cert", help="Path to CA certificate")
```

Comment:
This is path to a CA certificate file or CA certificate folder or both? Could add a little more text to describe it?